website/src/components/ConformanceShowcase/styles.module.css

```diff
@@ -0,0 +1,27 @@
+.conformanceShowcase {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin: auto;
+}
+
+.conformanceShowcaseItem {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    text-align: left;
+}
+
+.conformanceImage img {
+    max-width: 25rem;
```

Comment:
Instead of setting both max-width and min-width to the same value, couldn't you just set `width: 25rem;`?

Reply:
no, apparently these things have a different meaning in flex box